src/layout/layout_entity.cpp

```
@@ -28,13 +28,30 @@ namespace moth_ui {
+        case LayoutEntityType::Group:
+            return std::make_unique<LayoutEntityGroup>(nullptr);
         default:
             log::error("Unknown layout entity type: {}", magic_enum::enum_name(type));
             assert(false && "unknown entity type");
             return nullptr;
         }
     }
 
+    std::unique_ptr<LayoutEntity> LoadEntity(nlohmann::json const& json, LayoutEntityGroup* parent, LayoutEntity::SerializeContext const& context) {
+        LayoutEntityType type = json.value("type", LayoutEntityType::Unknown);
+        std::unique_ptr<LayoutEntity> entity = CreateLayoutEntity(type);
```

Clang (14.0.6)

[error] variable 'entity' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)